Library/Formula/cctools.rb

```diff
@@ -11,6 +11,7 @@ class Cctools < Formula
   end
 
   bottle do
+    does_not_need_relocation
     cellar :any
```

Comment:
Also: this should be autodetected in `bottle.rb` or it'll be overwritten any upgrade/rebottling